scripts/testing_scripts/cm_test_all_sandia

```diff
@@ -502,12 +502,12 @@ elif [ "$MACHINE" = "kokkos-dev-2" ]; then
 
   SKIP_HWLOC=True
 
-  BASE_MODULE_LIST="sems-env,sems-cmake/3.12.2,kokkos-env,kokkos-hwloc/1.10.1/base,sems-<COMPILER_NAME>/<COMPILER_VERSION>"
-  GCC91_MODULE_LIST="sems-env,sems-cmake/3.12.2,kokkos-env,kokkos-hwloc/1.10.1/base,<COMPILER_NAME>/<COMPILER_VERSION>"
-  NVCC_MODULE_LIST="sems-env,sems-cmake/3.12.2,kokkos-env,kokkos-hwloc/1.10.1/base,<COMPILER_NAME>/<COMPILER_VERSION>,sems-gcc/7.3.0"
+  BASE_MODULE_LIST="sems-env,kokkos-env,kokkos-hwloc/1.10.1/base,sems-<COMPILER_NAME>/<COMPILER_VERSION>"
```

Comment:
I'm not sure that removing sems-cmake/3.12.2 works here.  The script does a module purge before loading the modules for the selected build, and the default cmake is 2.7 or something like that....Kokkos needs at least 3.10 for the cmake build process.

Reply:
@jeffmiles63 at line 500 sems-cmake/3.12.2 is loaded for all builds after the purge, so we won't have to worry about a version of cmake that is unsupported.